Service/ClientFinderInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Service;
+
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+
+interface ClientFinderInterface
```

Comment:
@toniperic Since we know this is going to be apart of our public API, please add comments in the class header that describe the interface's purpose (same goes for the method itself).

Also, can we just put `@api` on the entire interface instead of an individual method?

Reply:
Yeah, moved `@api` to the interface.

However I'm not sure how I'd additionally describe such contract, could you maybe help?

It finds clients, as the contract suggests, so adding comments without actually providing any additional information seems a bit redundant to me?